WcaOnRails/app/models/competition.rb

```diff
@@ -807,4 +821,20 @@ def serializable_hash(options = nil)
       organizers: organizers,
     }
   end
+
+  # See https://github.com/thewca/worldcubeassociation.org/wiki/WCA-Competition-JSON-Format
+  def big_json
+    big_json_db = JSON.parse(super || "{}")
+    big_json_db.merge(
+      "formatVersion" => "WCA Competition 0.3",
+      "competitionId" => id,
+      "scrambleProgram" => nil,
+      "resultsProgram" => nil,
+    )
+  end
+
+  def big_json=(new_big_json)
+    big_json = self.big_json.merge(new_big_json)
+    super(JSON.generate(big_json))
```

Comment:
Wouldn't this enable an organizer to create a bigjson that doesn't match the competition's name/events by changing the big json and importing it again?

Earlier we talked about a merge where some of the competition fields would be overridden by the ones in the database. Maybe I'm missing something but I think we just export the bigjson in the db without ensuring some of its data are accurate.
I think any params is permitted for the bigjson update right now, maybe we could "blacklist" those meant to be changed only through the website, and merge the real value?
(Or maybe we want to do so later on?)